pkgs/development/tools/rust/humility/default.nix

```
@@ -12,7 +12,7 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "humility";
-  version = "unstable-2023-11-08";
+  version = "unstable-2025-02-25";
```

```suggestion
  version = "0-unstable-2025-02-25";
```
There is a new policy that all versions must start with a digit